Makefile

```diff
@@ -138,7 +138,7 @@ $(DOCKER_BUILD_TARGETS):
 	$(eval DOCKER_BUILD_CMD := $(subst docker_build_,,$@))
 	docker build \
 		$(DOCKER_BUILD_FLAGS) \
-		-t $(REGISTRY)/$(APP_NAME)-$(DOCKER_BUILD_CMD):$(BUILD_TAG) \
+		-t $(REGISTRY)/$(APP_NAME)-$(DOCKER_BUILD_CMD)-$(GOARCH):$(BUILD_TAG) \
```

Comment:
You also need to change [this line](https://github.com/jetstack/cert-manager/blob/46efa7879c5c0a29f164af25cb99d7523a0aaeff/Makefile#L149) to use:

```
$(REGISTRY)/$(APP_NAME)-$(DOCKER_PUSH_CMD)-$(GOARCH):$(BUILD_TAG)
```

as the source image tag to be used when tagging the image before pushing.

If you leave the second parameter as-is for now, we can then continue to push the same image tags. We can then follow up with another PR to enable actually pushing alternate arch images to the docker registry, once it is supported.